draft-ietf-tls-tls13.md

```diff
@@ -2116,6 +2128,9 @@ hash
   SHA-224, SHA-256, SHA-384, and SHA-512 {{SHS}}, respectively.  The
   "none" value is provided for future extensibility, in case of a
   signature algorithm which does not require hashing before signing.
+  The usage of MD5 and SHA-224 are deprecated. The md5_RESERVED and
+  sha224_RESERVED values MUST NOT be offered or negotiated by any
+  implementation.
```

Comment:
I have a line doing that in my WIP branch at the moment. We can get this bit in for now, though.